packages/e2e-tests/specs/change-detection.test.js

```diff
@@ -223,9 +224,9 @@ describe( 'Change detection', () => {
 		// Keyboard shortcut Ctrl+S save.
 		await pressKeyWithModifier( 'primary', 'S' );
 
-		await releaseSaveIntercept();
-
 		await assertIsDirty( true );
+
+		await releaseSaveIntercept();
```

Comment:
Any reason for swapping these?

Reply:
>Should prompt if content added without save

The save should come after the assertion.